api/models/ivr_prompt_mapping.py

```diff
@@ -15,4 +15,4 @@ class IvrPromptMapping(TimestampMixin, db.Model):
     ivr_prompt_id = db.Column(db.Integer, db.ForeignKey("ivr_prompt.id"))
     mapped_table_name = db.Column(db.String(255))
     mapped_table_column_name = db.Column(db.String(255))
-    default_value = db.Column(db.String(255))
+    default_value = None
```

Comment:
How it will work? What is the use of `None` value while defining the model structure

Reply:
Okay, getting your point. Didn't self reviewed it yet.